### ISSUE TITLE
[FEATURE REQUEST] discourage frosted

### ISSUE THREAD

**Issue by a-takahashi223** [PyCQA/prospector#501]
**Is your feature request related to a problem? Please describe.**
`frosted` does not work with python 3.10.
That's natural because `frosted` is deprecated for some years.
But when you install `prospector[with_everything]`, `frosted` will be installed. Then if you run `prospector -w frosted`, you will get errors.
Thus `with_everything` is discouraged.

**Describe the solution you'd like**
Where `with_everything` is introduced in the documentation,  annotate to recommend `with_pyroma,with_vulture,with_mypy,with_bandit` instead as `frosted` is deprecated.

**Describe alternatives you've considered**
Stop supporting `frosted`.
This is not good because backward compatibility will be broken.

**Comment by Pierre-Sassoulas** [comment]
```diff
- frosted = {version = ">=1.4.1", optional = true}
+ frosted = {version = ">=1.4.1;python_version<"3.10"", optional = true}
``` 
Could works, right ?

**Comment by a-takahashi223** [comment]
Oh, sorry. I seem to have missed the point.
`frosted` is deprecated and should not be used in new projects.
I am suggesting we tell that in the documentation.

**Comment by carlio** [comment]
Yes, I already have plans to remove frosted in versions 2+ and to add deprecation warnings starting with 1.8

In the meantime, I will be able to add the additional python version filter (as @Pierre-Sassoulas suggested) along with explicit documentation that `frosted` will not work in 3.10, and since those are optional tools it's possible to continue running even with `-w frosted` but emit an error/warning.

Frosted is definitely going to be removed, I just need to do it carefully to be backwards compatible.
